src/pages/index.js

```diff
@@ -6,173 +7,241 @@ import PopupWithForm from "../components/PopupWithForm.js";
-const imagePreviewModal = document.querySelector("#image-modal");
-const modalImage = imagePreviewModal.querySelector("#modal-image");
-const imageTitle = imagePreviewModal.querySelector("#modal-image-title");
-const imagePreviewCloseButton = document.querySelector("#image-modal-close");
 const imageAddButton = document.querySelector(".profile__add-button");
 const imageAddModal = document.querySelector("#add-modal");
-const imageAddCloseButton = imageAddModal.querySelector("#add-modal-close");
-const cardListEL = document.querySelector(".cards__list");
+// const cardListEL = document.querySelector(".cards__list");
 
-// Classes
-const section = new Section(
-  {
-    items: initialCards,
-    renderer: (cardData) => {
-      section.addItem(cardData);
-    },
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "02e2a34c-53b2-49ac-946d-3d9791bbdd65",
+    "Content-Type": "application/json",
   },
-  ".cards__list"
-);
+});
 
-const newCardPopup = new PopupWithForm("#add-modal", handleAddCardSubmit);
+// Initial Start Up
 
 const userInfo = new UserInfo({
   profileName: ".profile__title",
   jobElement: ".profile__description",
+  avatar: ".profile__image",
 });
 
-const profileEditPopup = new PopupWithForm(
-  "#edit-modal",
-  handleProfileEditFormSubmit
-);
+api
+  .getUserInfo()
+  .then((res) => {
+    userInfo.setUserInfo({
+      name: res.name,
+      job: res.about,
+      avatar: res.avatar,
+    });
+  })
+  .catch((err) => {
+    console.log(err);
+  });
+
+// Image Popup
 
 const popupImage = new PopupWithImage({
   popupSelector: "#image-modal",
 });
 
-newCardPopup.setEventListeners();
-profileEditPopup.setEventListeners();
 popupImage.setEventListeners();
 
-profileEditCloseButton.addEventListener("click", () => {
-  profileEditPopup.close();
-});
-
 // Profile Edit Modal
-function openProfileEditForm() {
+
+const profileEditPopup = new PopupWithForm(
+  "#edit-modal",
+  handleProfileEditFormSubmit
+);
+
+profileEditPopup.setEventListeners();
+
+function fillProfileForm() {
   const profileInfo = userInfo.getUserInfo();
   profileTitleInput.value = profileInfo.name;
   profileDescriptionInput.value = profileInfo.job;
-  profileEditPopup.open();
 }
 
-profileEditButton.addEventListener("click", openProfileEditForm);
+profileEditButton.addEventListener("click", () => {
+  fillProfileForm();
+  profileEditPopup.open();
+});
 
-function handleProfileEditFormSubmit(inputValues) {
-  const profileInfo = {};
-  profileInfo.name = inputValues.title;
-  profileInfo.job = inputValues.description;
-  console.log(profileInfo);
-  userInfo.setUserInfo(profileInfo);
+function handleProfileEditFormSubmit(profileInfo) {
+  profileEditPopup.setLoading(true);
+  api
+    .updateUserInfo(profileInfo)
+    .then((res) => {
+      userInfo.setUserInfo({
+        name: res.name,
+        job: res.about,
+      });
+    }, profileEditPopup.close())
```

Comment:
`    }, profileEditPopup.close())
`

It doesn't work like that

You can't have `,` (a comma) here

You should do that in `then` (inside the then function)

```js

    .updateUserInfo(profileInfo)
    .then((res) => {
      userInfo.setUserInfo({
        name: res.name,
        job: res.about,
      });
      profileEditPopup.close()
    })
    .catch((err) => {
```